lib/stats/stats-registry.c

```diff
@@ -51,31 +67,65 @@ stats_unlock(void)
 }
 
 static StatsCluster *
-_grab_cluster(gint stats_level, const StatsClusterKey *sc_key, gboolean dynamic)
+_grab_dynamic_cluster(const StatsClusterKey *sc_key)
 {
   StatsCluster *sc;
 
-  if (!stats_check_level(stats_level))
-    return NULL;
+  sc = g_hash_table_lookup(stats_cluster_container.dynamic_clusters, sc_key);
+  if (!sc)
+    {
+      if (!stats_check_dynamic_clusters_limit(_number_of_dynamic_clusters()))
+        return NULL;
+      sc = stats_cluster_dynamic_new(sc_key);
+      _insert_cluster(sc);
+      if ( !stats_check_dynamic_clusters_limit(_number_of_dynamic_clusters()))
```

Comment:
unnecessary space here

Reply:
Hmm... we have a tool that  checks style and blocks merge until we fix format issues.
So I wouldn't fix it here (fixing astyle would be better as currently we don't check this kind of format issue and it is possible that there are other parts of the code which is affected by the same problem).